Auto/install.sh

```diff
@@ -0,0 +1,65 @@
+echo - Starting Install -
```

Comment:
can you include an ALv2 license header here? As an example, see: http://svn.apache.org/repos/asf/oodt/trunk/pcs/core/src/main/bin/pcs_trace